sdk/cosmos/azure-cosmos/docs/StatusCodes.md

```diff
@@ -0,0 +1,53 @@
+|403|1014||`Forbidden/LogicalPartitionExceedsStorage` - indicates that the data size of a logical partition exceeds the service quota (currently 20 GB). See [TSG](https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-forbidden#non-data-operations-are-not-allowed) for more details|
+|404|0||`Not found` - Indicates that the resource the client tried to read does not exist (on the replica being contacted). Depending on the consistency level used this could be a transient error condition - but when using less than strong consistency one that the application needs to be able to handle gracefully. See [TSG](https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-not-found) for more details.|
+|404|1002|(x)|`Not Found/Read session no available` - Indicates that a client uses session consitency and reached a replica that has a replication lag and has not caught-up to the requested session token. In many cases this error condition will be transient. But there are certain situation in which it could persist for longer period of times - either a wrong session token is being provided in the application or in a Multi-Write region set-up operations are regulary directed to different regions|
+|404|1003|(x)|`Not Found/Owner resource does not exist` - Indicates that a client attempted to process an operation on a resource whose parent does not exist. For example an attempt to do a point operation on a document when the container does not exist (yet). Can be transient when attempting document operations immediately after creating a container etc. - but when not transient usually means a bug in your application.|
+|404|1024|x|`Not Found/Incorrect Container resource id` - Indicates that a client attempted to use a container that has recently been deleted and rectreated. So, the cached container id in the client is stale - and identifies the previosuly deleted container. The SDK will trigger retries - in general applications need to be able to tolerate that conatiner deletion and immediate recreation will take up-to a few seconds/minutes to be replicated across all regions.|
+|408|\*|x|`Request timeout` - Indicates a tiemout for an attempted operation. See [TSG](https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-java-sdk-request-timeout) for more details.|
+|408|20008|(x)|`Request timeout/End-to-end timeout exceeded` - Indicates that the application defined end-to-end timeout was exceeded when processing an operation. This will usually be a transient error condition - exceptions are when the application defines unrealistis end-to-end timeouts - for example when executing a query that could very well take a few seconds because it is relatively inefficient or when the end-to-end timeout is lower than the to-be-expected network transit time between the application's location and the Cosmso DB sevice endpoint.|
```

Comment:
```suggestion
|408|20008|(x)|`Request timeout/End-to-end timeout exceeded` - Indicates that the application defined end-to-end timeout was exceeded when processing an operation. This will usually be a transient error condition - exceptions are when the application defines unrealistic end-to-end timeouts - for example when executing a query that could very well take a few seconds because it is relatively inefficient or when the end-to-end timeout is lower than the to-be-expected network transit time between the application's location and the Cosmos DB service endpoint.|
```

Reply:
Fixed in next iteration